service/frontend/workflow_handler.go

```diff
@@ -421,6 +421,21 @@ func (wh *WorkflowHandler) StartWorkflowExecution(ctx context.Context, request *
 	return &workflowservice.StartWorkflowExecutionResponse{RunId: resp.GetRunId(), EagerWorkflowTask: resp.GetEagerWorkflowTask()}, nil
 }
 
+func (wh *WorkflowHandler) unaliasedSearchAttributesFrom(
```

Comment:
I would remove word `From`:
```suggestion
func (wh *WorkflowHandler) unaliasedSearchAttributes(
```

Reply:
Next commit.